proposed/runtime-binding.md

```diff
@@ -205,20 +205,37 @@ We will offer a configuration knobs for hosts that enable artificial roll-forwar
 
 The host exposes the following configuration knobs that you can use to control binding behavior:
 
+### Version
+
+The `version` knob specifies the minimum (or floor) version required by an application as a 3-part version. 
+
+`version` can be set in the following ways:
+
+* `RuntimeFrameworkVersion` in an MSBuild project
+* `version` in `.runtimeconfig.json`
+* `--fx-version` via the CLI
+
+
+If `RuntimeFrameworkVersion` is not set, then `version` defaults to the `0` patch for the specified target framework. For example, `version` would be set to `3.0.0` by default for the `netcoreapp3.0` target framework.
+
+MSBuild project settings are only consulted at build-time and are then written as properties to `.runtimeconfig.json`, which can be consulted at runtime.
+
 ### RollForward
 
-We will introduce a new knob, called `RollForward`, with the following values:
+`RollForward` specifies the roll-forward policy for an application, either as a fallback in case a specific version 
+
+`RollForward` can have the following values:
 
-* `None` -- Do not roll forward. Only bind to specified version.
-* `Patch` -- Roll forward to the highest patch verison. This disables minor version roll forward.
-* `Minor` -- Roll forward to the lowest higher minor verison, if requested minor version is missing. If the requested minor version is present, then the `Patch` policy is used.
-* `Major` -- Roll forward to lowest higher major version, and lowest minor version, if requested major version is missing. If the requested major version is present, then the `Minor` and `Patch` policies are used, as appropriate.
-* `AlwaysLatestMinor` -- Roll forward to latest minor version, even if requested minor version is present. Intended for COM hosting scenario.
-* `AlwaysLatestMajor` -- Roll forward to latest major and highest minor version, even if requested major is present. Intended for COM hosting scenario.
+* `LatestPatch` -- Roll forward to the highest patch verison. This disables minor version roll forward.
+* `Minor` -- Roll forward to the lowest higher minor verison, if requested minor version is missing. If the requested minor version is present, then the `LatestPatch` policy is used.
+* `Major` -- Roll forward to lowest higher major version, and lowest minor version, if requested major version is missing. If the requested major version is present, then the `Minor` policy is used.
+* `LatestMinor` -- Roll forward to latest minor version, even if requested minor version is present. Intended for COM hosting scenario.
```

Comment:
Not sure we should explicitly state that this is for COM... it is useful in other cases as well. But if you think it makes sense, I'm fine with it.